### DISCUSSION TITLE
I want use text-shadow, thankyou!

### DISCUSSION THREAD

**Discussion by bd-dxg** [tailwindlabs/tailwindcss#9391] [Help] [answered]
Why doesn't Tailwindcss provide a text-shadow attribute?

I tried to add the text-shadow property through the configuration item, but it didn't seem to work.

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="stylesheet" href="./style.css" />
    <script src="https://cdn.tailwindcss.com"></script>
    <script defer>
      tailwind.config = {
        theme: {
          textShadow: {
            default: '0 2px 0 #000',
            md: 'black 0.1rem 0.1rem 0.2rem',
            h2: '0 0 3px #FF0000, 0 0 5px #0000FF',
            h1: '0 0 3px rgba(0, 0, 0, .8), 0 0 5px rgba(0, 0, 0, .9)',
          },
        },
      }
    </script>
    <title>Document</title>
  </head>
  <body class="box-border select-none">
    <main
      class="absolute font-serif top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 w-[300px] h-[300px] bg-[#bfa] grid place-content-center text-white text-shadow-md"
    >
      <section class="p-5 align-middle text-center text-9xl">A</section>
    </main>
  </body>
</html>
```

I use Tailwindcss in the cdn environment because it's a simple project and I don't want to use Tailwindcss CLI.

---
**version of Tailwind CSS:** v3.1.8
**version of Node.js:** v16.15.0
**using browser:** Microsoft Edge 105.0.1343.42  (arm64)
**using system:** macOS 12.5.1

**Comment by wongjn** [comment, marked as answer]
You can try using [arbitrary properties](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) when there is no utility class for CSS you'd like to use, for example, `[text-shadow:black_0.1rem_0.1rem_0.2rem]`

**Reply by bd-dxg** [reply]
emmm, Your answer is valid .
However, this is also too inelegant for Tailwindcss.
Nonetheless, I am very grateful for your answer, it is useful to me, thank you very much
